easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2023.12-iimkl-2023b.eb

```diff
@@ -0,0 +1,117 @@
+toolchainopts = {
+    'pic': True, 'lowopt': True, 'strict': True,
+    'oneapi': True, 'optarch': False,  # prevents -xHost
+}
+
+builddependencies = [
+    ('hypothesis', '6.90.0'),
+    ('UnZip', '6.0'),
+    # scipy >= 1.9.0 uses Meson/Ninja
+    ('Meson', '1.2.3'),
+    ('meson-python', '0.15.0'),
+    ('Ninja', '1.11.1'),
+    ('pkgconf', '2.0.3'),   # required by scipy
+]
+
+dependencies = [
+    ('Python', '3.11.5'),
+    ('Python-bundle-PyPI', '2023.10'),
+    ('pybind11', '2.11.1'),  # required by scipy
+]
+
+use_pip = True
+
+# order is important!
+exts_list = [
+    ('numpy', '1.26.2', {
+        'easyblock': 'PythonPackage',  # pip install builds numpy v1.26.x via spin/meson/ninja
+        'patches': [
+            'numpy-1.25.1_disable_fortran_callback_test.patch',
+            'numpy-1.25.1_disable-broken-test_long_long_map.patch',
+            'numpy-1.26.2_fix_selected_kind_for_ifort.patch',
+            'numpy-1.25.1_disable-broken-fortran-docstring-test.patch',
+            'numpy-1.25.1_fix-test_features.patch',
+        ],
+        'checksums': [
+            {'numpy-1.26.2.tar.gz': 'f65738447676ab5777f11e6bbbdb8ce11b785e105f690bc45966574816b6d3ea'},
+            {'numpy-1.25.1_disable_fortran_callback_test.patch':
+             '3c02bd9973b7082fde9f9d18edfeb05798226ccb5731a56f5677269200c345cf'},
+            {'numpy-1.25.1_disable-broken-test_long_long_map.patch':
+             'fa0fb0a16c4f1339a974c1c84b79df21dc9bfdc14e3e68f6aebaf5e30bad3fe9'},
+            {'numpy-1.26.2_fix_selected_kind_for_ifort.patch':
+             '3622bacbe4063373cc0b1c0aa0ffe3d30dfe132f6a763b15b45cc6387cbd51d7'},
+            {'numpy-1.25.1_disable-broken-fortran-docstring-test.patch':
+             '8a4d36e3b3a9c9bf43df6e5214f3883234a069b80c5c1027a7c84bd5cb133457'},
+            {'numpy-1.25.1_fix-test_features.patch':
+             '1c05ee5d105fe2f824416dd6dd5c64ed0c1cd710a002b4e6dbfafff19203adc5'},
+        ],
+    }),
+    ('ply', '3.11', {
+        'checksums': ['00c7c1aaa88358b9c765b6d3000c6eec0ba42abca5351b095321aef446081da3'],
+    }),
+    ('gast', '0.5.4', {
+        'checksums': ['9c270fe5f4b130969b54174de7db4e764b09b4f7f67ccfc32480e29f78348d97'],
+    }),
+    ('beniget', '0.4.1', {
+        'checksums': ['75554b3b8ad0553ce2f607627dad3d95c60c441189875b98e097528f8e23ac0c'],
+    }),
+    ('pythran', '0.14.0', {
+        'checksums': ['42f3473946205964844eff7f750e2541afb2006d53475d708f5ff2d048db89bd'],
+    }),
+    ('versioneer', '0.29', {
+        'checksums': ['5ab283b9857211d61b53318b7c792cf68e798e765ee17c27ade9f6c924235731'],
+    }),
+    ('scipy', '1.12.0', {
+        'enable_slow_tests': True,
+        'ignore_test_result': False,
+        'patches': [
+            'scipy-1.11.1_disable-tests.patch',
+            'scipy-1.11.1_xfail-aarch64_test_maxiter_worsening.patch',
+            'scipy-1.11.1_disable-tests-iimkl.patch',
+            'scipy-1.11.4_disable-test_branch_cut.patch',  # intel might need -assume=minus0
```

Comment:
Instead of this patch, how do I pass buildopts `-assume=minus0` to the compiler for this extension?

BTW setting `CFLAGS`  [is highly discouraged in meson](https://mesonbuild.com/Reference-tables.html#compiler-and-linker-flag-environment-variables)?

Reply:
Can you mention the Meson CFLAGS issue in https://github.com/easybuilders/easybuild-framework/issues/3331?

Reply:
Oh wait, I'm mixing up `CFLAGS` and `CPATH`, nevermind :D

Reply:
> Instead of this patch, how do I pass buildopts -assume=minus0 to the compiler for this extension?

You could probably do it via `preinstallopts`, with a construct like `export CFLAGS="$CFLAGS -assume=minus0" && ` in there, but I'm not sure that's a whole lot better.

A patch is pretty OK here I think?